apps/backend/src/s3-manager/s3-manager.service.ts

```diff
@@ -22,14 +22,11 @@ import { Injectable } from '@nestjs/common';
 export class S3ManagerService {
   public s3: S3Client;
   constructor() {
-    const config: S3ClientConfig =
-      process.env.NODE_ENV === 'production'
-        ? {
-            endpoint: process.env.AWS_ENDPOINT,
-            region: 'ca-central-1',
-            forcePathStyle: true,
-          }
-        : {};
+    const config: S3ClientConfig = {
+      endpoint: process.env.AWS_ENDPOINT,
```

Comment:
during local development AWS_ENDPOINT=='http://localhost:9000' (minio)

Reply:
This is currently showing as
> AWS_ENDPOINT='http://minio:9000'

In .env.example. Good idea to change?